package.json

```diff
@@ -18,9 +19,10 @@
     "generate": "stencil generate"
   },
   "dependencies": {
+    "@saasquatch/dom-context-hooks": "^0.1.0",
     "@stencil/core": "^1.17.4",
     "debug": "^4.3.1",
-    "dom-context": "^1.0.0",
+    "dom-context": "^1.0.1",
     "haunted": "^4.7.1"
```

Comment:
Can we please check if `stencil-hooks` actually builds with the latest version of haunted `4.8.0` as it will be installed with this specifier? In reference to this issue: https://github.com/matthewp/haunted/issues/248